app/views/carts/_quantity_field.html.haml

```diff
@@ -0,0 +1,6 @@
+.input-group
+  %div.input-link.dec.button{id: "dec-#{item.id}", onclick: "inc_dec('-', this)"}
+    %i.fa.fa-minus.line-height-40
+  = field.text_field :quantity, class: "order-items-#{item.id}-quantity form-control quantity-input", min: 1, value: "#{item.quantity}"
```

Comment:
you have css code for each order_item `order-items-#{item.id}-quantity` ????

Reply:
No, it's need to change value in quantity field by click + or -.

Reply:
Whut?oO
You need css class to change value?)
If you need some target for you javascript you need to use an id attribute but not a class. Classes are used ONLY for css.